src/style-library/components/molecules/fields/BoxTokenField.scss

```
@@ -0,0 +1,12 @@
+@use '../../../styles/mixins' as *;
+
+// `src/token-controls` stays host-agnostic, so `ControlShell` ships its label with layout only and no
+// type treatment. Here that label sits directly beside `FieldLabel`'s (NAME, and every other field in
+// the panel), so it has to read as the same kind of label — via the shared mixin rather than a copy of
+// its values.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)